### PR TITLE
[ci] report stylelint error on failure

### PR DESCRIPTION
**Pull request by kiranparajuli589**
- update `nextcloud` lint related dev dependencies
- install node and npm version from package.json in the ci
- fixed `eslint` and style lint inside `src`

### CODE REVIEW

**Comment by kiranparajuli589** [comment]
see style lint report for `css` and `vue` files here: https://github.com/nextcloud/integration_openproject/runs/5581830589?check_suite_focus=true

**Comment by kiranparajuli589** [comment]
still need more fixes

File: stylelint.config.js [hunk 1]
```diff
@@ -1,33 +1,3 @@
-module.exports = {
```
Comment (kiranparajuli589) [inline stylelint.config.js:1]: the same config is inherited from https://github.com/nextcloud/stylelint-config/blob/master/index.js

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -47,6 +41,22 @@ jobs:
           key: ${{ runner.os }}-composer-${{ hashFiles('**/composer.lock') }}
           restore-keys: ${{ runner.os }}-composer-
 
+      - name: Read package.json node and npm engines version
+        uses: skjnldsv/read-package-engines-version-actions@v1.1
+        id: versions
+        with:
+          fallbackNode: '^14'
+          fallbackNpm: '^6'
```
Comment (individual-it) [inline .github/workflows/ci.yml:49]: readme says, we need at least NPM 7

Comment (kiranparajuli589) [reply .github/workflows/ci.yml]: fixed

**Review by individual-it — APPROVED** [review]

**Pull request merged**